### ISSUE TITLE
Improve UX

### ISSUE THREAD

**Issue by satyajitnayk** [satyajitnayk/mock-data#1]
Suggestions by @tiholic
1. Make the RHS update data automatically without using the generate button.
2. And copy the text upon clicking the RHS box.

**Comment by satyajitnayk** [comment]
The RHS data should be a plain JSON. Currently it shows:
```
[
{"a": "value"},
{"b": "value"},
]
```

It should show:
```
{"a": "value", "b": "value"}
```

**Comment by satyajitnayk** [comment]
addressed by 7976154efaaa37e73b092fb88bd1f6c6e4467676, bcf8d3b88a32b43edaf4500a3cea64ed8d2fcd4c
